lib/util.js

```diff
@@ -72,9 +72,9 @@ const getConfigFileContents = config => {
 Host ${config.host}
   HostName ${config.hostName}
   User git
+  IgnoreUnknown UseKeychain
```

Comment:
Moving this up when writing to ssh/config file because in non MacOS systems we get errors if don't define `IgnoreUnknown UseKeychain` before defining `UseKeyChain yes` in ssh config file